source/query/queries.txt

```diff
@@ -1,25 +1,69 @@
-.. _past-query-view:
-
-=================
-View Past Queries
-=================
+=======================================
+Managing Saved Queries and Aggregations
+=======================================
 
 .. default-domain:: mongodb
 
-Compass automatically stores up to 20 most recent queries for each
-collection. To view these queries, click the history icon in the query bar.
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Starting in |compass| 1.31, you can load saved queries and aggregations from 
+the :guilabel:`My Queries` view once you are connected to your cluster.
+This page explains how to add and view favorite queries and 
+aggregations.
+
+Saving an Aggregation Pipeline
+------------------------------
+
+.. include:: /includes/fact-save-pipeline.rst
+
+.. _favorite-queries:
+
+Saving a Favorite Query
+-----------------------
+
+You can favorite a query so that you can find it later. If you
+load a favorite query, you can change it without
+changing the original saved copy.
+
+To add a query to your favorites:
 
-.. figure:: /images/compass/query-history-view.png
+.. include:: /includes/steps/saving-favorite-queries.rst
+
+Viewing Favorite Queries
+------------------------
+
+Starting in |compass| 1.31, you can view your saved queries
+and aggregation pipelines on the :guilabel:`My Queries` view 
+once connected to your cluster.
+
+.. _saved-query-view:
+
+Using the :guilabel:`My Queries` View
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you click a saved or favorite query tile, Compass opens the
+:guilabel:`Documents` tab with the filter loaded.
+
+When you click a saved or favorite pipeline tile, Compass opens the 
+:guilabel:`Aggregations` tab with the pipeline loaded.
+
+.. figure:: /images/compass/query-my-queries-splash.png
    :figwidth: 720 px
-   :alt: Query history view
+   :alt: My queries view
+
+You can also view favorite queries from the :guilabel:`Favorites` tab 
```

Comment:
This should be in its own sub-section

Reply:
Updated.

Reply:
I don't see this as updated. We want a sub-section titled something like "Using the FavoritesTab" for this content"